cytosnake/common/errors.py

```diff
@@ -95,6 +95,10 @@ class BarcodeMissingError(BaseFileNotFound):
     """Raised when a barcode file is required"""
 
 
+class PlateDataNotFoundError(BaseFileNotFound):
```

Comment:
I may have missed it but wanted to double check: where does this exception come into use within this PR?

Reply:
You're correct. I did not use it. Removed!